src/components/grid/GridTitelInfo.vue

```diff
@@ -1,11 +1,21 @@
 <template>
-  <div class="detail-info-hover" >
-    <div class="detail-info">
-      {{variableName(info)}}
-    </div>
-    <span class="variable-title">
-      {{variableName(info)}}
+  <div class="grid-titel">
+    <span class="variable-title" :id="`variable-${name}`">
+      {{variableName}}
     </span>
+    <b-popover
+      :target="`variable-${name}`"
+      triggers="hover click"
+      :title="name">
+      <dl>
+        <dt>Question (in English)</dt>
```

Comment:
We can already show variable code and name sins we have it in local state.

Reply:
(this.label and this.name)

Reply:
Name is the title, Trynke does not list label as one of the things for the popup.
Shall we add label anyways, you think?